event-sourcing/event-repository/event-repository-jpa/src/main/java/uk/gov/justice/services/eventsourcing/repository/jpa/eventlog/EventLogRepositoryJdbc.java

```diff
@@ -0,0 +1,170 @@
+package uk.gov.justice.services.eventsourcing.repository.jpa.eventlog;
+
+import uk.gov.justice.services.eventsourcing.repository.jpa.exception.EventLogRepositoryException;
+import uk.gov.justice.services.eventsourcing.repository.jpa.exception.InvalidSequenceIdException;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+/**
+ * Implementation of the {@link EventLogRepository} interface.
+ */
+public class EventLogRepositoryJdbc implements EventLogRepository {
+
+    /**
+     * Column Names
+     */
+    static final String PRIMARY_KEY_ID = "id";
+    static final String COL_STREAM_ID = "stream_id";
+    static final String COL_SEQUENCE_ID = "sequence_id";
+    static final String COL_NAME = "name";
+    static final String COL_METADATA = "metadata";
+    static final String COL_PAYLOAD = "payload";
+
+    static final long INITIAL_VERSION = 0L;
+
+    static final String JNDI_APP_NAME_LOOKUP = "java:app/AppName";
+    static final String JNDI_DS_EVENT_STORE_PATTERN = "java:/app/%s/DS.EventStore";
+
+    /**
+     * Statements
+     */
+    static final String SQL_FIND_BY_STREAM_ID = "select * from event_log where stream_id=?";
+    static final String SQL_FIND_BY_STREAM_ID_AND_SEQUENCE_ID = "select * from event_log where stream_id=? and sequence_id>=?";
+    static final String SQL_FIND_LATEST_SEQUENCE_ID = "select max(sequence_id) from event_log where stream_id=?";
+    static final String SQL_INSERT_EVENT_LOG = "insert into event_log (id, stream_id, sequence_id, name, metadata, payload ) " +
```

Comment:
We don't have a coding standard defined for SQL, but I would like to have all keywords (SELECT, INT< FROM, WHERE, etc) all upper case for clarity.